scripts/generated_file_list.txt

```diff
@@ -0,0 +1,10 @@
+accounts/abi/bind/precompilebind/precompile_config_template.go
+accounts/abi/bind/precompilebind/precompile_config_test_template.go
+accounts/abi/bind/precompilebind/precompile_contract_template.go
+accounts/abi/bind/precompilebind/precompile_contract_test_template.go
+accounts/abi/bind/precompilebind/precompile_event_template.go
+accounts/abi/bind/precompilebind/precompile_module_template.go
+precompile/allowlist/event.go
+precompile/contracts/feemanager/event.go
+precompile/contracts/nativeminter/event.go
+precompile/contracts/rewardmanager/event.go
```

Comment:
again we can just use avalanche header in these files right? (these are generated but we can ignore that header)